test/configuration_store_test.py

```diff
@@ -1,41 +1,39 @@
-from eppo_client.configuration_requestor import (
-    AllocationDto,
-    ExperimentConfigurationDto,
-)
+from eppo_client.models import Flag
 from eppo_client.configuration_store import ConfigurationStore
+from eppo_client.models import VariationType
 
-test_exp = ExperimentConfigurationDto(
-    subject_shards=1000,
-    enabled=True,
-    name="randomization_algo",
-    allocations={"allocation-1": AllocationDto(percent_exposure=1, variations=[])},
-)
 
 TEST_MAX_SIZE = 10
 
-store: ConfigurationStore[ExperimentConfigurationDto] = ConfigurationStore(
-    max_size=TEST_MAX_SIZE
+store: ConfigurationStore[str] = ConfigurationStore(max_size=TEST_MAX_SIZE)
+mock_flag = Flag(
+    key="mock_flag",
+    variation_type=VariationType.STRING,
+    enabled=True,
+    variations={},
+    allocations=[],
+    total_shards=10000,
 )
 
 
 def test_get_configuration_unknown_key():
-    store.set_configurations({"randomization_algo": test_exp})
+    store.set_configurations({"flag": mock_flag})
     assert store.get_configuration("unknown_exp") is None
 
 
 def test_get_configuration_known_key():
-    store.set_configurations({"randomization_algo": test_exp})
-    assert store.get_configuration("randomization_algo") == test_exp
+    store.set_configurations({"flag": mock_flag})
+    assert store.get_configuration("flag") == mock_flag
 
 
 def test_evicts_old_entries_when_max_size_exceeded():
```

Comment:
What's this? Do we have a max size concept elsewhere?

Reply:
Not sure -- I did not change the logic here